src/app/graphql/queries/get-organization-addresses.graphql

```diff
@@ -0,0 +1,27 @@
+query getOrganizationAddresses($id: String!, $count: Int, $cursor: String, $sort: String) {
+  organization(id: $id) {
+    addresses(first: $count, after: $cursor, sort: $sort) {
+      totalCount
+      edges {
+        cursor
+      }
+      items {
+        firstName
+        lastName
+        line1
+        phone
+        email
+        countryCode
+        postalCode
+        regionName
+        city
+      }
+      pageInfo {
+        endCursor
+        hasNextPage
+        hasPreviousPage
+        startCursor
```

Comment:
Does we really need this information? I don't see this fields are used anywhere in code. Please left only needed fields in this query

Reply:
done